pci/src/class.rs

```diff
@@ -292,4 +413,18 @@ mod test {
             Ok(Class::MassStorage(MassStorage::Sata(iface::Sata::Achi1))),
         );
     }
+
+    #[test]
+    fn test_display() {
+        assert_eq!(
+            Class::MassStorage(MassStorage::Sata(iface::Sata::Achi1)).to_string(),
+            "Mass Storage Controller : Serial ATA : AHCI 1.0"
```

Comment:
I'd prefer to not put the spaces on both sides of the colon (which feels a little weird IMO). What do you think about maybe something like
```rust
"Mass Storage Controller: Serial ATA (AHCI 1.0)"
```
we could also consider using the alt format specifier, or the precision specifier, to control how much information is formatted?